### ISSUE TITLE
How can I get access to the 64 * 64 model weight?

### ISSUE THREAD

**Issue by StarNextDay** [claroche-r/FastDiffusionEM#7]
Excuse me. I really appreciate your work and would like to do some experiments based on it. However, while searching for the weights of the 64 × 64 kernel denoiser, I could only find the 33 × 33 version in the Google Drive repository. I noticed in another issue that you already have a 64 × 64 version, so I was wondering if you could provide the corresponding weights or tell me on how to train such a denoiser myself. Looking forward to your response.

**Comment by claroche-r** [comment]
Thanks for your interest. Unfortunately, I removed the 64x64 as it was not working as expected and I did not have time to search for better architecture. Working with larger kernels will require a bigger network with a larger receptive field. I believe you can use a similar architecture as [BlindDPS](https://openaccess.thecvf.com/content/CVPR2023/papers/Chung_Parallel_Diffusion_Models_of_Operator_and_Image_for_Blind_Inverse_CVPR_2023_paper.pdf) which uses a transformer (so that the receptive field is of the size of the image). 

To train the model, you can use motion blur kernels from [motionblur](https://github.com/LeviBorodenko/motionblur). 
Once you have your architecture, you just need to train it as a regular denoiser (loss = MSE(net(x + noise) - x)).

**Comment by StarNextDay** [comment]
Thank you for your suggestions! I'll try it later.
